src/util.ts

```diff
@@ -79,74 +79,75 @@ export function getBuyURL(networkCode = '1', address?: string, amount = 5) {
  * @param fromBlock? - Block from which transactions are needed
  * @returns - URL to fetch the transactions from
  */
-export function getEtherscanApiUrl(networkType: string, address: string, fromBlock?: string): string {
+export function getEtherscanApiUrl(
+  networkType: string,
```

Comment:
This is a pre-existing issue, but the network controller actually exports a type that could be used here to improve the typing: `NetworkType`.

Which has brought to my attention that this would break completely for the network types "localhost" and "rpc" :thinking: